src/Workspaces/Core/Portable/EmbeddedLanguages/VirtualChars/VirtualChar.cs

```diff
@@ -23,33 +25,57 @@ namespace Microsoft.CodeAnalysis.EmbeddedLanguages.VirtualChars
     /// </summary>
     internal readonly struct VirtualChar : IEquatable<VirtualChar>
     {
-        public readonly char Char;
+        /// <summary>
+        /// Can represent any unicode code point.  Values are represented in Utf32.
+        /// </summary>
+        public readonly uint CodePoint;
```

Comment:
❔ Can we represent this as `System.Text.Rune` instead?

Reply:
that would make the most sense.  however, i don't know how to use that at the Workspace layer (which targets core 2.0).  Is that possible?